client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -92,11 +92,7 @@ function ReaderSubscriptionListItem( {
 	);
 
 	return (
-		<div
-			className={ classnames( 'reader-subscription-list-item', className, {
-				'has-email-settings': showEmailSettings && isFollowing,
```

Comment:
Couldn't find any reference to `has-email-settings` in stylesheets, so I've removed it.